src/pages/home/ReportScreen.js

```diff
@@ -221,7 +221,7 @@ class ReportScreen extends React.Component {
                     )}
                 >
                     <FullPageNotFoundView
-                        shouldShow={!this.props.report.reportID}
+                        shouldShow={!this.props.report.reportID && !this.props.report.isLoadingReportActions}
```

Comment:
This prevents the No Access screen from showing while the `OpenApp` call is loading